src/app/projects/components/project-settings/components/embeddings/embeddings.component.ts

```diff
@@ -0,0 +1,155 @@
+import { Component, Input, OnInit, SimpleChanges } from '@angular/core';
+import { Subscription } from 'rxjs';
+import { first } from 'rxjs/operators';
+import { Project } from 'src/app/base/entities/project';
+import { ProjectApolloService } from 'src/app/base/services/project/project-apollo.service';
+import { Attribute } from '../../entities/attribute.type';
+import { DownloadedModel } from '../../entities/downloaded-model.type';
+import { Embedding } from '../../entities/embedding.type';
+import { DataHandlerHelper } from '../../helper/data-handler-helper';
+import { SettingModals } from '../../helper/modal-helper';
+
+@Component({
+  selector: 'kern-embeddings',
+  templateUrl: './embeddings.component.html',
+  styleUrls: ['./embeddings.component.scss'],
+})
+export class EmbeddingsComponent implements OnInit {
+
+  @Input() project: Project;
+  @Input() attributesArrayTextUsableUploaded: Attribute[];
+  @Input() settingModals: SettingModals;
+  @Input() isManaged: boolean;
+  @Input() embeddings: Embedding[];
+  @Input() embeddingHandlesMap: { [key: string]: any };
```

Comment:
Still has Map in the name. Maybe just embeddingHandles or embeddingHandlesDict?

Also the syntax for "any kind of key" is [key:string]:any to represent that the dictionary is a string key and holds something. 

This is not wrong but to better understand the code I recently changed the variable name of key to be a bit more representative -> in this case `[embeddingId:string]:any` so the developer knows that its not "any" key but specifically an embeddingId. 

Again not wrong but might help a dev in a few months to better understand the code :)